src/components/booksList.js

```diff
@@ -1,24 +1,42 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import { RemoveBook } from '../actions';
+import { RemoveBook, ChangeFilter } from '../actions';
 import Book from './book';
+import CategoryFilter from './CategoryFilter';
 
 const mapStateToProps = state => ({
   books: state.books,
+  filter: state.filter,
 });
 
 const mapDispatchToProps = dispatch => ({
   removeBook: book => dispatch(RemoveBook(book)),
+  changeFilter: category => dispatch(ChangeFilter(category)),
 });
```

Comment:
**OPTIONAL**: The [recommended](https://react-redux.js.org/using-react-redux/connect-mapdispatch#two-forms-of-mapdispatchtoprops) way to use `mapDispatchToProps` is using the object form.